x-pack/solutions/security/plugins/security_solution/public/app/routes.tsx

```diff
@@ -47,7 +48,9 @@ const PageRouterComponent: FC<RouterProps> = ({ children, history }) => {
           <Routes>
             <Route path="/">
               <CasesContext owner={[APP_ID]} permissions={userCasesPermissions}>
-                <HomePage>{children}</HomePage>
+                <ReactQueryClientProvider>
```

Comment:
Cases context has it's own queryClient which we need to re-override, but need to test this behavior. This is primarily useful just for our testing purposes